packages/snap/src/snap/hedera.ts

```diff
@@ -0,0 +1,149 @@
+import { PrivateKey } from '@hashgraph/sdk';
```

Comment:
This file should not even exist because there is already a connectHederaAccount under rpc/hedera. Please revert these changes as this is not needed @dchagastelles